### DISCUSSION TITLE
Potential bug in `useTreeState`

### DISCUSSION THREAD

**Discussion by airtable-JayRansijn** [adobe/react-spectrum#1598] [Q&A]
I am trying to use `useTreeState` from https://react-spectrum.adobe.com/react-stately/useTreeData.html and finding that `tree.move()` doesn’t always seem to re-render, and it seems the `parentKey` is incorrect sometimes.

CodeSandbox: https://codesandbox.io/s/treeview-for-issue-u73ne?file=/src/App.tsx 
Example: https://u73ne.csb.app/

**Comment by devongovett** [comment]
Possibly related: https://github.com/adobe/react-spectrum/pull/1465
